plugins/labhub.py

```diff
@@ -193,6 +185,26 @@ def callback_message(self, msg):
                 self.send(msg.frm, response)
                 self.hello_world_users.add(user)
 
+        mentioned = []
+        room_members = msg.frm.room.occupants
```

Comment:
We need to fetch the people once reducing the number of API calls and eventually update the list as soon as there are new people in the room, will improve this later on once we've fixed backend.